### DISCUSSION TITLE
Remove Figma EX

### DISCUSSION THREAD

**Discussion by Andreslav** [Moonvy/figmaEX#11] [Q&A]
If I want to uninstall the FigmaEX-Installer-win app, how do I do it?

**Comment by yArna** [comment]
`2. Restore Backup` option in the FigmaEX installer

```node
select command （选择一个操作） (Use arrow keys)
  1. Start Install（安装） 
❯ 2. Restore Backup（还原备份） 
```

**Reply by Andreslav** [reply]
Maybe I didn't understand you, but I want to delete FigmaEX, not restore the backup ..

![изображение](https://user-images.githubusercontent.com/39444579/115950135-24174100-a4e2-11eb-9fe3-136748af541b.png)

**Reply by Andreslav** [reply]
@yArna Figma stops launching if files are deleted app-97.7.0\resources\*, app-96.4.0\resources\*
Your app is interesting and useful, but there should be a way to remove it painlessly.

**Reply by yArna** [reply]
The FigmaEX installation modifies the figma files and backs up the original files, so 'Restore Backup' allows you to go back to a time when figmaEX was not installed.

**Reply by yArna** [reply]
We updated the installer [FigmaEX v1.0.3](https://github.com/Moonvy/figmaEX/releases/tag/v1.0.3) , should be able to uninstall normally with him

**Reply by Andreslav** [reply]
> The FigmaEX installation modifies the figma files and backs up the original files, so 'Restore Backup' allows you to go back to a time when figmaEX was not installed.

It's not obvious...

> We updated the installer [FigmaEX v1.0.3](https://github.com/Moonvy/figmaEX/releases/tag/v1.0.3) , should be able to uninstall normally with him

@yArna, thanks! It's much better this way.
